main.js

```diff
@@ -0,0 +1,107 @@
+// Add event listener 
+// prevent default and const for inputs
+document.getElementById('form-post').addEventListener('submit', function(e) {
+  e.preventDefault();
+
+ 
+  const textFromPost = document.getElementById('text-from-post').value;
+  const writerPosted = document.getElementById('writer-posted').value;
+  
+  postToAdd(textFromPost, writerPosted);
+
+  
```

Comment:
watch for all these empty spaces
```suggestion
  const textFromPost = document.getElementById('text-from-post').value;
  const writerPosted = document.getElementById('writer-posted').value;
  postToAdd(textFromPost, writerPosted);
```